pandas/_libs/algos.pyx

```diff
@@ -2,10 +2,12 @@
 
 import cython
 from cython import Py_ssize_t
+from cython.parallel import prange
 
 from libc.stdlib cimport malloc, free
 from libc.string cimport memmove
 from libc.math cimport fabs, sqrt
+from cpython cimport bool
```

Comment:
does cython treat this differently from built-in bool?

Reply:
I believe this is the type used by cython to declare a python boolean (an object which means many operations can only be taken on it when gil is taken).